images/smb-csi-driver-operator.yml

```
@@ -0,0 +1,36 @@
+content:
+  source:
+    dockerfile: Dockerfile.samba
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/csi-operator.git
+      web: https://github.com/openshift/csi-operator
+    ci_alignment:
+      streams_prs:
+        ci_build_root:
+          stream: rhel-9-golang-ci-build-root
+        auto_label:
+        - approved
+        - lgtm
+distgit:
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
+  component: smb-csi-driver-operator-container
+  bundle_component: smb-csi-driver-operator-bundle-container
+enabled_repos:
+- rhel-9-baseos-rpms
+- rhel-9-appstream-rpms
+for_payload: false
+from:
+  builder:
+  - stream: rhel-9-golang
+  member: openshift-enterprise-base-rhel9
+name: openshift/ose-smb-csi-driver-rhel9-operator
+name_in_bundle: smb-csi-driver-operator
+owners:
+- aos-storage-staff@redhat.com
+update-csv:
+  bundle-dir: stable
+  manifests-dir: config/samba/manifests/
+  registry: image-registry.openshift-imageregistry.svc:5000
+  valid-subscription-label: '["OpenShift Container Platform", "OpenShift Platform Plus"]'
```

```suggestion
  valid-subscription-label: '["OpenShift Kubernetes Engine", "OpenShift Container Platform", "OpenShift Platform Plus"]'
```
(assuming it should be the same as ose-secrets-store-csi-driver-operator?)